CybridSDK/Components/Accounts/View/AccountTransfersViewModel.swift

```diff
@@ -45,6 +45,20 @@ class AccountTransfersViewModel: NSObject {
             }
         }
     }
+
+    internal static func getAmountOfTransfer(_ transfer: TransferBankModel) -> String {
+
+        let amount = transfer.state == .completed ? transfer.amount ?? 0 : transfer.estimatedAmount ?? 0
+        let amountValue = CDecimal(amount)
+        let amountFormatted = AssetFormatter.forBase(Cybrid.fiat, amount: amountValue)
```

Comment:
What is `Cybrid.fiat`? The transfer (in the case of book transfers) could be crypto (e.g. `BTC`).